oauth2_http/java/com/google/auth/oauth2/ServiceAccountCredentials.java

```diff
@@ -340,6 +343,23 @@ public AccessToken refreshAccessToken() throws IOException {
     HttpRequest request = requestFactory.buildPostRequest(new GenericUrl(tokenServerUri), content);
     request.setParser(new JsonObjectParser(jsonFactory));
 
+    request.setIOExceptionHandler(new HttpBackOffIOExceptionHandler(new ExponentialBackOff()));
+    request.setUnsuccessfulResponseHandler(
+        new HttpBackOffUnsuccessfulResponseHandler(new ExponentialBackOff()).setBackOffRequired(
```

Comment:
It looks like for `ExponentialBackOff` the default elapsed time will be 15 minutes and default max interval 60 seconds. Both of them (especially the first one) seem very high. Please confirm that it is ok. If it is not, please override the default values.